libs/network/include/network/p2pservice/p2p_http_interface.hpp

```diff
@@ -155,7 +155,7 @@ class P2PHttpInterface : public http::HTTPModule
       block["hash"]         = "0x" + b->hash.ToHex();
       block["previousHash"] = "0x" + b->previous_hash.ToHex();
       block["merkleHash"]   = "0x" + b->merkle_hash.ToHex();
-      block["miner"]        = b->miner.display();
+      block["miner"]        = b->miner_id.identifier().ToBase64();
```

Comment:
This needs to be the address (block explorer needs it to be)

Reply:
Ok